src/operations/graph/graphHelpers.js

```diff
@@ -929,7 +944,7 @@ class GraphHelper {
                 scope: requestInfo.scope,
                 isUser: requestInfo.isUser,
                 patientIdsFromJwtToken: requestInfo.patientIdsFromJwtToken,
-                args: Object.assign({'base_version': base_version}, {'id': idList}), // add id filter to query
+                args: Object.assign(args, {'id': idList, 'resource': null}), // add id filter to query
```

Comment:
is this the line that's making us do the deepCopy of args and pass around both args and originalArgs? could we have constructQueryAsync deepCopy and use a modified version of args internally, and let the rest of the codebase use an immutable args?